lute/fs/src/fs.cpp

```diff
@@ -70,6 +72,24 @@ const char* UV_DIRENT_TYPES[] = {
     UV_TYPENAME_BLOCK,
 };
 
+static UVFile* getFileHandle(lua_State* L, int index)
+{
+    if (!lua_islightuserdata(L, index))
+    {
+        luaL_errorL(L, "Error: expected file handle");
+        return nullptr;
+    }
```

Comment:
```suggestion
    if (!lua_islightuserdata(L, index))
        luaL_errorL(L, "Error: expected file handle");
```

`luaL_errorL` is noret, no?

Reply:
Got rid of the return nullptr.